packages/server/server.ts

```diff
@@ -45,6 +46,7 @@ uws
   .get('/sse/*', SSEConnectionHandler)
   .get('/sse-ping', SSEPingHandler)
   .get('/self-hosted/*', selfHostedHandler)
+  .get('/jira-attachements/:fileName', jiraImagesHandler)
```

Comment:
*attachments (remove the extra e)

Reply:
🤦 I've just installed https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker so hopefully that won't happen again! 😂